.github/ISSUE_TEMPLATE/project.md

```diff
@@ -21,35 +21,56 @@ assignees: ''
 
 <!-- Please fill out the following questions to rationalize the prioritization of this project -->
 
-- Does this serve our mission?
+- Does this serve our mission
 
 - Will this help to make us financially sustainable within a year?
 
-- Is it feasible to build and maintain with our team and runway?
-
 - Can this make us move faster?
 
-- Does this increase quality and not technical debt?
+- Does this increase quality? 
+
+- Does this add technical debt?
+
+- Is it feasible to build and maintain with our current team and runway?
 
-- Will this bring us more Collective and financial contributors?
+- Is this problem already solved elsewhere or is it based on insight or research?
```

Comment:
I think it's important that we are able to prioritse projects by things we think are a good idea, things we suspect to be a good idea or things we know to be a good idea :D

Reply:
Sounds to me like `Is this problem already solved elsewhere` should be a separate question than knowing what are the research or insight that motivate this project.

Reply:
```suggestion
- What insight or research is this project based on?
- Is thisproblem already solved elsewhere?
```